examples/data-sources/powermax_port/variables.tf

```diff
@@ -0,0 +1,19 @@
+# variable "username" {
+#   type = string
+# }
+
+# variable "password" {
+#   type = string
+# }
+
```

Comment:
@doriac11 Please uncomment before merging the PR.

Reply:
Done, did it on the other PR forgot about this one.